docs/md/melange_convert_apkbuild.md

```diff
@@ -15,14 +15,14 @@ Converts an APKBUILD package into a melange.yaml
 
 Converts an APKBUILD package into a melange.yaml.
 
-```
-melange convert apkbuild [flags]
+```sh
+$ melange convert apkbuild [flags]
```

Comment:
As above, we elide the `$` where we can in order to allow users to copy and paste.